src/core/manager.c

```diff
@@ -3788,6 +3940,11 @@ int manager_reload(Manager *m) {
 
         lookup_paths_log(&m->lookup_paths);
 
+        /* The only canonical reference left to the dynamically allocated parts of these BPF programs is
+         * going to be on the other side of manager_deserialize, so the freeable parts can now be freed. The
+         * program itself will be detached as part of manager_vacuum. */
+        manager_skeletonize_all_cgroup_bpf_programs(m);
```

Comment:
Not sure, but this seems to do nothing, no? As `m->bpf_limbo_progs` is `NULL` here?